README.md

```diff
@@ -1,24 +1,125 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <img src="https://user-images.githubusercontent.com/93520178/203661672-538bff13-8797-4d20-9b82-372aa011d464.png" alt="logo" width="100"  height="auto" />
+  <br/>
 
-* Ruby version
+</div>
 
-* System dependencies
+# 📗 Table of Contents
 
-* Configuration
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
 
-* Database creation
+# 📖 Firmtecs_blog <a name="about-project"></a>
 
-* Database initialization
+> This Firmtecs_blog app will be a classic example of a blog website. I created a fully functional website that will show the list of posts and encourage readers to interact with them by adding comments and liking posts.
 
-* How to run the test suite
+Please click on the video below 👇 for more explanation
 
-* Services (job queues, cache servers, search engines, etc.)
+## Explanation video
 
-* Deployment instructions
+[]()
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://www.postgresql.org/">Postgres</a></li>
+  </ul>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- IDE or code editor: **VsCode, atom, sublime**...
+- To have [Ruby](https://www.ruby-lang.org/en/), [RoR gem](https://rubyonrails.org/) and [Git](https://git-scm.com/) installed on your system.
+
+### Setup
+
+Clone this project on your local machine using the following command on your terminal:
+
+```sh
+  git clone https://github.com/blas147/Firmtecs_blog.git
+```
+
+### Install
+
+Run the following command to install all project's dependencies:
+
+```sh
+bundle install
+```
+
+Run the following command to install Rspec for tests:
+
+```sh
+gem install rspec
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails s
+```
+
+## Run Tests
+
+Run the following command to catch all style errors:
+
+```sh
+rubocop
+```
+
+Run the following command for the rspec tests(with details):
+
+```sh
+rspec ./spec --format documentation
+
+```
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Chukwuma Paul Mosanya**
+
+[GitHub](https://github.com/blase147) | [Twitter](https://twitter.com/ChukwumaMosanya) | [LinkedIn](https://www.linkedin.com/in/chukwuma-mosanya)
+
```

Comment:
### **Optional**

- [ ] Please add section for `Future Features` in your [README](https://github.com/microverseinc/readme-template) file.